src/Service.Tests/CosmosTests/SchemaGeneratorTest.cs

```diff
@@ -78,9 +78,9 @@ public void TestSchemaGeneratorUsingMultipleJson(string jsonFilePath, string gql
             }
 
             string actualSchema = SchemaGenerator.Generate(jArray, "planet", baseConfig);
-            string expectedSchema = File.ReadAllText($"{gqlFilePath}/{gqlFileName}");
+            string expectedSchema = File.ReadAllText($"{gqlFilePath}/{gqlFileName}").Trim();
```

Comment:
were some operations adding extra whitespace? Does needing to add `Trim()` here indicate some issue where output isn't consistent?